### ISSUE TITLE
XML file contains lines with duplicate attribute k

### ISSUE THREAD

**Issue by JFritzle** [IgorMagellan/Tiramisu#4]
Lines 2684, 2687 and 2688 of file Tiramisu.xml in Tiramisu 4.2 contain duplicate attribute **k**, e.g.
`<caption cat="tms_landnames" k="name" ... k="name" ... />`
in line 2684.

This causes Java error messages and exceptions like
```
ParseError at [row,col]:[2684,190]
Message: http://www.w3.org/TR/1999/REC-xml-names-19990114#AttributeNotUnique?caption&k
javax.xml.stream.XMLStreamException: ParseError at [row,col]:[2684,190]
Message: http://www.w3.org/TR/1999/REC-xml-names-19990114#AttributeNotUnique?caption&k
	at com.sun.org.apache.xerces.internal.impl.XMLStreamReaderImpl.next(Unknown Source)
	at com.sun.xml.internal.stream.XMLEventReaderImpl.nextEvent(Unknown Source)
	at ...
```

**Comment by IgorMagellan** [comment]
These and similar bugs have already been fixed in the current version of Tiramisu(4.3), not in Release 4.2. 
Thanks for pointing out possible errors anyway.

**Comment by JFritzle** [comment]
Confirmed.
